File: string-paradigm-expression-no-wrapper/src/main/java/com/steelzack/string/paradigms/nowrapper/expression/RegionSpellingCheck.java
package com.steelzack.string.paradigms.nowrapper.expression;

import java.util.List;

public class RegionSpellingCheck extends SpellingCheckBase{



	@SuppressWarnings("unchecked")
	public List<RegionImpl> getSpellingDoubleCommaRegions() {
		return (List<RegionImpl>)(List<?>)spellingDoubleCommaRegions;
	}

	@SuppressWarnings("unchecked")
	public List<RegionImpl> getSpellingWhiteSpacesRegions() {
		return (List<RegionImpl>)(List<?>)spellingWhiteSpacesRegions;
	}

	@SuppressWarnings("unchecked")
	public List<RegionImpl> getSpellingBeginCapitalRegions() {
		return (List<RegionImpl>)(List<?>)spellingBeginCapitalRegions;
	}
}

File: string-paradigm-expression-no-wrapper/src/main/kotlin/org/jesperancinha/string/paradigms/nowrapper/expression/RegionSpellingCheck.kt
package org.jesperancinha.string.paradigms.nowrapper.expression

class RegionSpellingCheck : SpellingCheckBase() {
    override val spellingDoubleCommaRegions: List<RegionImpl>?
        get() = spellingDoubleCommaRegions as List<*> as List<RegionImpl>
    override val spellingWhiteSpacesRegions: List<RegionImpl>?
        get() = spellingWhiteSpacesRegions as List<*> as List<RegionImpl>
    override val spellingBeginCapitalRegions: List<RegionImpl>?
        get() = spellingBeginCapitalRegions as List<*> as List<RegionImpl>
}